features/list-style.yml.dist

```diff
@@ -0,0 +1,306 @@
+  - css.properties.list-style-type.hangul
+  - css.properties.list-style-type.hangul-consonant
+  - css.properties.list-style-type.urdu
+
+  # baseline: false
+  # support:
+  #   chrome: "45"
+  #   chrome_android: "45"
+  #   edge: "79"
+  #   safari: "17"
+  #   safari_ios: "17"
+  - css.properties.list-style-type.ethiopic-halehame
+  - css.properties.list-style-type.ethiopic-halehame-am
+
+  # baseline: false
+  # support:
+  #   chrome: "86"
+  #   chrome_android: "86"
+  #   edge: "86"
+  #   firefox: "68"
+  #   firefox_android: "68"
+  - css.selectors.marker
+
+  # baseline: false
+  # support:
+  #   chrome: "86"
+  #   chrome_android: "86"
+  #   edge: "86"
+  #   firefox: "80"
+  #   firefox_android: "80"
+  - css.selectors.marker.animation_and_transition_support
+
+  # baseline: false
+  # support:
+  #   chrome: "91"
+  #   chrome_android: "91"
+  #   edge: "91"
+  #   safari: "1"
+  #   safari_ios: "1"
+  - css.properties.list-style-type.upper-greek
+
+  # baseline: false
+  # support:
+  #   chrome: "91"
+  #   chrome_android: "91"
+  #   edge: "91"
+  #   safari: "5"
+  #   safari_ios: "4.2"
+  - css.properties.list-style-type.afar
+  - css.properties.list-style-type.amharic
+  - css.properties.list-style-type.amharic-abegede
+  - css.properties.list-style-type.binary
+  - css.properties.list-style-type.ethiopic
+  - css.properties.list-style-type.ethiopic-abegede
+  - css.properties.list-style-type.ethiopic-abegede-am-et
+  - css.properties.list-style-type.ethiopic-abegede-gez
+  - css.properties.list-style-type.ethiopic-abegede-ti-er
+  - css.properties.list-style-type.ethiopic-abegede-ti-et
+  - css.properties.list-style-type.ethiopic-halehame-aa-er
+  - css.properties.list-style-type.ethiopic-halehame-aa-et
+  - css.properties.list-style-type.ethiopic-halehame-am-et
+  - css.properties.list-style-type.ethiopic-halehame-gez
+  - css.properties.list-style-type.ethiopic-halehame-om-et
+  - css.properties.list-style-type.ethiopic-halehame-sid-et
+  - css.properties.list-style-type.ethiopic-halehame-so-et
+  - css.properties.list-style-type.ethiopic-halehame-tig
+  - css.properties.list-style-type.lower-hexadecimal
+  - css.properties.list-style-type.lower-norwegian
+  - css.properties.list-style-type.octal
+  - css.properties.list-style-type.oromo
+  - css.properties.list-style-type.sidama
+  - css.properties.list-style-type.somali
+  - css.properties.list-style-type.tigre
+  - css.properties.list-style-type.tigrinya-er
+  - css.properties.list-style-type.tigrinya-er-abegede
+  - css.properties.list-style-type.tigrinya-et
+  - css.properties.list-style-type.tigrinya-et-abegede
+  - css.properties.list-style-type.upper-hexadecimal
+  - css.properties.list-style-type.upper-norwegian
+
+  # baseline: false
+  # support:
+  #   chrome: "91"
+  #   chrome_android: "91"
+  #   edge: "91"
+  #   safari: "5.1"
+  #   safari_ios: "5"
+  - css.properties.list-style-type.asterisks
+  - css.properties.list-style-type.footnotes
```

Comment:
`asterisks` and `footnotes` do not appear in the spec- should they be present here?

Reply:
Both are marked as `standard_track: true` in BCD. If we drop them, then we ought to contribute an issue/PR upstream to fix that.

Reply:
This was a can of worms. There are 43 items here not included in the spec, due to a quirk in the collector. I've opened https://github.com/mdn/browser-compat-data/issues/24545 to address that. In the meantime, I'm thinking I should remove all the keys here that are not in the spec, correct?

Reply:
Yes, let's set aside the keys that are not in the spec and return to them later.